package.json

```diff
@@ -28,14 +28,17 @@
     "url": "https://github.com/QuantStack/jupytercad.git"
   },
   "scripts": {
-    "build": "jlpm build:schema && jlpm run build:lib && jlpm run build:worker && jlpm run build:labextension:dev",
+    "build": "jlpm build:schema && jlpm run build:lib && jlpm run build:copy-opencascade && jlpm run build:worker && jlpm run build:labextension:dev",
     "build:schema": "json2ts -i src/schema -o src/_interface --no-unknownAny --unreachableDefinitions --cwd ./src/schema && cd src/schema && node ../../schema.js",
     "build:prod": "jlpm run clean && jlpm build:schema && jlpm run build:worker:prod && jlpm run build:lib && jlpm run build:labextension",
     "build:labextension": "jupyter labextension build .",
     "build:labextension:dev": "jupyter labextension build --development True .",
     "build:lib": "tsc",
+    "build:copy-opencascade": "copyfiles -f src/worker/opencascade/*.js src/worker/opencascade/*.d.ts src/worker/opencascade/*.wasm lib/worker/opencascade",
```

Comment:
We're using the `CopyPlugin` in `extension.webpack.config.js` to copy `wasm` files, maybe we only need to update the config in this file instead of adding a copy command here.